lib/util.js

```diff
@@ -517,7 +517,7 @@ function formatCollectionIterator(ctx, value, recurseTimes, visibleKeys, keys) {
   var nextRecurseTimes = recurseTimes === null ? null : recurseTimes - 1;
   var vals = mirror.preview();
   var output = [];
-  for (let o of vals) {
+  for (const o of vals) {
```

Comment:
`const` for a loop variable is completely valid - the binding is const inside the loop body and updated for each loop iteration. The non-strict behavior of `const` in V8 is a holdover from its original non-ES6 implementation of the feature.

Spidermonkey's error is from them not having implemented the feature.